personality/cream_ce/monitor.pan

```diff
@@ -43,22 +43,24 @@ type cream_monitoring = {
 
 bind '/software/components/metaconfig/services/{${CREAM_MONITORING_CONF_FILE}}/contents' = cream_monitoring;
```

Comment:
wait, this works? never seen that before...

Reply:
Yes, it works :+1:

Reply:
heh, i didn't know that. in that case it's really a pity that prefix doesn't support it also.